deploy/podman/quadlet/README.md

```
@@ -10,7 +10,7 @@ This directory contains an example of how to use a rootless Podman pod defined t
    ```bash
    echo -n "YOUR PASSWORD" | podman secret create grimmory_db_pass -
    ```
-4. (Optional) `podman pull ghcr.io/grimmory-tools/grimmory:v0.38.2` to pre-pull the image
+4. (Optional) `podman pull ghcr.io/grimmory-tools/grimmory:latest` to pre-pull the image
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 3; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)